e2e-tests/playwright/tests/functional/system_console/system_users/actions.spec.ts

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {TestBrowser} from '@e2e-support//browser_context';
+import {Client, createRandomTeam, createRandomUser} from '@e2e-support/server';
+import {expect, test} from '@e2e-support/test_fixture';
+import {components} from '@e2e-support/ui/components';
+import {SystemConsolePage} from '@e2e-support/ui/pages/system_console';
+import {getRandomId} from '@e2e-support/util';
+import {UserProfile} from '@mattermost/types/users';
+
+/**
+ * Setup a new random user, and search for it such that it's the first row in the list
+ * @param pw
+ * @param pages
+ * @returns A function to get the refreshed user, and the System Console page for navigation
+ */
+async function setupAndGetRandomUser(
+    pw: {testBrowser: TestBrowser; initSetup: () => Promise<{adminUser: UserProfile | null; adminClient: Client}>},
+    pages: {SystemConsolePage: typeof SystemConsolePage},
+) {
+    const {adminUser, adminClient} = await pw.initSetup();
+
+    if (!adminUser) {
+        throw new Error('Failed to create admin user');
+    }
+
+    // # Log in as admin
+    const {page} = await pw.testBrowser.login(adminUser);
+
+    // # Create a random user to edit for
+    const user = await adminClient.createUser(createRandomUser(), '', '');
+    const team = await adminClient.createTeam(createRandomTeam());
+    await adminClient.addToTeam(team.id, user.id);
+
+    // # Visit system console
+    const systemConsolePage = new pages.SystemConsolePage(page);
+    await systemConsolePage.goto();
+    await systemConsolePage.toBeVisible();
+
+    // # Go to Users section
+    await systemConsolePage.sidebar.goToItem('Users');
+    await systemConsolePage.systemUsers.toBeVisible();
+
+    // # Search for user-1
+    await systemConsolePage.systemUsers.enterSearchText(user.email);
+    const userRow = await systemConsolePage.systemUsers.getNthRow(1);
+    await userRow.getByText(user.email).waitFor();
+    const innerText = await userRow.innerText();
+    expect(innerText).toContain(user.email);
+
+    return {getUser: () => adminClient.getUser(user.id), systemConsolePage};
+}
+
+test('MM-T5520-1 should activate and deactivate users', async ({pw, pages}) => {
+    const {getUser, systemConsolePage} = await setupAndGetRandomUser(pw, pages);
```

Comment:
can 1 user be used in all tests in a spec file? 
Different user in Different specs would make sense when we run in specs in parallel.